src/mygrad/tensor_base.py

```diff
@@ -879,7 +901,7 @@ def T(self):
 
 
 # set all comparison operators - mirrors ndarray methods
-def tensor_to_array_wrapper(func):
+def tensor_to_array_wrapper(func: Callable) -> Callable:
```

Comment:
It's not always worth specifying argument types, but specifying the return types of these functions could be useful.